lib/match/rules.js

```diff
@@ -85,6 +85,7 @@ module.exports = {
 
 	'date'		: validator.isDate,
 	'datetime': validator.isDate,
+	'time': function(x) {return moment(x, "hh:mm:ss a").isValid();},
```

Comment:
This breaks consistency between `date`, `datetime` and `time` is that intentional?